nengo/spa/bind.py

```diff
@@ -0,0 +1,39 @@
+import nengo
+from nengo.spa.module import Module
+
+
+class Bind(Module):
+    """A module for binding together both inputs
+
+    Parameters
+    ----------
+    dimensions : int
+        Number of dimensions for the two vectors to be compared
+    vocab : Vocabulary, optional
+        The vocabulary to use to interpret the vectors
+    n_neurons : int
+        Number of neurons to use in each product computation
+    invert_a, invert_b : bool
+        Whether to reverse the order of elements in either
+        the first input (`invert_a`) or the second input (`invert_b`).
+        Flipping the second input will make the network perform circular
+        correlation instead of circular convolution.
+    input_magnitude : float
+        Effective input magnitude for the multiplication.
+        The actual input magnitude will be this value times sqrt(2)
```

Comment:
It's not clear to me what this means, especially given the new product system.  What do you mean by "actual input magnitude"?  What multiplication?  What is this for?  What use case would there be for this parameter?

One possible solution here is just to remove the second line, since as far as I can tell it's both unclear and false.  As I said elsewhere, I'd also be supportive of just removing this parameter altogether, since I can't think of a spa use case for it.  But if the parameter is supposed to be here then it should be clear what it means.